### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Gloix** [vikmeup/SCLAlertView-Swift#402]
There is no compatibility with VoiceOver, so this library cannot be used in projects with accessibility for visually impaired in mind (almost every project).
Primarily, the elements inside the dialog (or even the dialog itself) are not focusable for VoiceOver. There is not even a single sound queue indicating the presence of a dialog on screen (that sound queue should be issued by VoiceOver itself).

**Comment by nfranke** [comment]
See this from the Objective C port of this library:

https://github.com/dogo/SCLAlertView/pull/261
